Server/Hubs/MainHub.cs

```diff
@@ -1,16 +1,28 @@
 ﻿using Microsoft.AspNetCore.SignalR;
 using System.Text.Json;
+using System.Collections.Generic;
 using System.Threading.Tasks;
+using System.Linq;
+using System;
 
 namespace SignalRServer.Hubs
 {
     public class MainHub : Hub
     {
+        public override Task OnConnectedAsync()
+        {
+            Console.WriteLine(Context.ConnectionId);
+            return base.OnConnectedAsync();
+        }
 
         public async Task SendPayloadA(string payload)
         {
-            var data = JsonSerializer.Deserialize<dynamic>(payload);
-            string json = JsonSerializer.Serialize(data);
+            var msg = "We are in A...";
+            Console.WriteLine(msg);
+            List<object> objects = new List<object>(){msg};
+            string json = JsonSerializer.Serialize(objects);
+            // var data = JsonSerializer.Deserialize<dynamic>(payload);
+            // string json = JsonSerializer.Serialize(data);
```

Comment:
It looks like this was changed for debugging... should revert before merging to continue using payload.

Reply:
Im working on it... Im so sorry about be late, but these days Im a little bit busy...